source/addonAPIVersion.py

```diff
@@ -13,12 +13,13 @@
 """
 
 CURRENT = (buildVersion.version_year, buildVersion.version_major, buildVersion.version_minor)
-BACK_COMPAT_TO = (2022, 1, 0)
+BACK_COMPAT_TO = (2023, 1, 0)
 """
 As BACK_COMPAT_TO is incremented, the changed / removed parts / or reasoning should be added below.
 These only serve to act as a reminder, the changelog should be consulted for a comprehensive listing.
 EG: (x, y, z): Large changes to speech.py
 ---
+(2023, 1, 0): speech as str was dropped in favor of only SpeechCommand, and security changes.

```

Comment:
This might be a bit hasty, given this change is not yet prepared.
Honestly, I'm not sure these summaries provide much value.
Developers would be much better placed to read the changes file.

Reply:
This PR will remain in a draft state until the 2022.1 API is ready to be frozen for beta.
I think we can update when we have a clearer picture on the release, I just aimed to capture what is currently proposed on the milestone.

Reply:
>Honestly, I'm not sure these summaries provide much value.
>Developers would be much better placed to read the changes file.

The comment above the summary suggests developers read the changes, but argues that the summaries are helpful. Should we change the suggested practice and drop the summaries?

Reply:
> > Honestly, I'm not sure these summaries provide much value.
> > Developers would be much better placed to read the changes file.
> 
> The comment above the summary suggests developers read the changes, but argues that the summaries are helpful. Should we change the suggested practice and drop the summaries?

IMO the current practice is OK. A developer still needs to read the change log. But the summary helps to remind the main API changes of each step.